app/Actions/PromptForCustomization.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Actions;
+
+use App\Support\BaseAction;
+
+class PromptForCustomization extends BaseAction
+{
+    /**
+     * Prompts the user for runtime customization.
+     *
+     * @return void
+     */
+    public function __invoke(): void
+    {
+        $customizeQuestion = 'Would you like to (R)un with current config, or (C)ustomize? Or (E)xit.';
+
+        $answer = false;
```

Comment:
Could you replace the complex contains with this:

```php
while (! in_array($answer, ['c', 'r', 'e'])) {
    $answer = strtolower($this->console ...etc);

    if ($answer === 'e') {
    // ...
```

This whole things reads kinda confusing, but I'm assuming it has something to do with how long-running Zero questions work.